### ISSUE TITLE
[TB] Atoms - Input Background - WCAG 4.1.2: Ensures every ARIA input field has an accessible name (.css-av774z-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input)

### ISSUE THREAD

**Issue by kburk1997** [finos/a11y-theme-builder#952]
### Issue
`Ensures every ARIA input field has an accessible name` ([aria-input-field-name](https://accessibilityinsights.io/info-examples/web/aria-input-field-name))
### Target application
[A11y ThemeBuilder](http://localhost:3000/designSystem/My%20Design)
### Element path
.css-av774z-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input
### Snippet
`<div tabindex="0" role="combobox" aria-controls=":r2j:" aria-expanded="false" aria-haspopup="listbox" class="MuiSelect-select MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input css-av774z-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-in...`
### How to fix
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
### Environment
Microsoft Edge version 126.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.43.0 (axe-core 4.8.4), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
### Screenshot
![Screenshot of color picker without proper label](https://github.com/finos/a11y-theme-builder/assets/11298269/c98c3862-a826-4969-b999-182b0fdeefad)

## Proposed Solution
Add `aria-labelledby="id of text"` (preferred, as it will programmatically associate input with text) or `aria-label="Overlay Color"` to fix the issue

**Comment by kburk1997** [comment]
This stems from the ColorSelect component - should be easy fix similar to #953

**Comment by kburk1997** [comment]
PR merged - closing
